dirty_cat/minhash_encoder.py

```diff
@@ -0,0 +1,148 @@
+
```

Comment:
Add a small docstring to describe what is in this file (to help other future developers)